Week1/homework/world-queries.txt

```diff
@@ -37,4 +37,179 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
+| SLV  | El Salvador                      | North America | Central America |    21041.00 |      1841 |    6276000 |           69.7 |   11863.00 |   11203.00 | El Salvador                         | Republic                                     | Francisco Guillermo Flores Pï¿½rez               |     645 | SV    |
+| SPM  | Saint Pierre and Miquelon        | North America | North America   |      242.00 |      NULL |       7000 |           77.6 |       0.00 |       NULL | Saint-Pierre-et-Miquelon            | Territorial Collectivity of France           | Jacques Chirac                                   |    3067 | PM    |
+| TCA  | Turks and Caicos Islands         | North America | Caribbean       |      430.00 |      NULL |      17000 |           73.3 |      96.00 |       NULL | The Turks and Caicos Islands        | Dependent Territory of the UK                | Elisabeth II                                     |    3423 | TC    |
+| TTO  | Trinidad and Tobago              | North America | Caribbean       |     5130.00 |      1962 |    1295000 |           68.0 |    6232.00 |    5867.00 | Trinidad and Tobago                 | Republic                                     | Arthur N. R. Robinson                            |    3336 | TT    |
+| USA  | United States                    | North America | North America   |  9363520.00 |      1776 |  278357000 |           77.1 | 8510700.00 | 8110900.00 | United States                       | Federal Republic                             | George W. Bush                                   |    3813 | US    |
+| VCT  | Saint Vincent and the Grenadines | North America | Caribbean       |      388.00 |      1979 |     114000 |           72.3 |     285.00 |       NULL | Saint Vincent and the Grenadines    | Constitutional Monarchy                      | Elisabeth II                                     |    3066 | VC    |
+| VGB  | Virgin Islands, British          | North America | Caribbean       |      151.00 |      NULL |      21000 |           75.4 |     612.00 |     573.00 | British Virgin Islands              | Dependent Territory of the UK                | Elisabeth II                                     |     537 | VG    |
+| VIR  | Virgin Islands, U.S.             | North America | Caribbean       |      347.00 |      NULL |      93000 |           78.1 |       0.00 |       NULL | Virgin Islands of the United States | US Territory                                 | George W. Bush                                   |    4067 | VI    |
++------+----------------------------------+---------------+-----------------+-------------+-----------+------------+----------------+------------+------------+-------------------------------------+----------------------------------------------+--------------------------------------------------+---------+-------+
+37 rows in set (0.00 sec)
+
+4.What's the top 10 countries by SurfaceArea ?
```

Comment:
This indeed was not asking for any column in particular... i should change it to name and surface area